index.js

```diff
@@ -0,0 +1,71 @@
+const menuContainer = document.getElementById('toolbar');
+const iconImage = menuContainer.querySelector('img');
+let controller = false;
+menuContainer.addEventListener('click', () => {
```

Comment:
- [ ] [OPTIONAL] Greate works with the hamburger menu however when I click the (X) button on mobile view and change the view to desktop, the Navbar is gon. kindly make sure to fix this bug so the nav bar is displayed all the time on the desktop view

Reply:
Thank you for your feedback, here you just have to refresh the page.